### ISSUE TITLE
Ensure we handle multiple chains properly

### ISSUE THREAD

**Issue by ethanfrey** [Lay3rLabs/WAVS#138]
Not like a config where you can run a server with testnet or localnode and it will always connect to that machine (which we have already)

But where you store chain info along with the services, and may run services that related to different chains on one wasmatic node. 
So you can add one service to localnode and one to testnet and they will run in parallel.

Additionally, it should support querying one chain and submitting to another, but that might need to be broken out to a new issue
